scripts/deprecation_report.py

```diff
@@ -0,0 +1,10 @@
+import sys
+from ganga.GangaCore.Utility.Deprecation import generate_deprecation_json_report
+
+root = "./ganga"
```

Comment:
This should be dynamically extracted from the code that is executed. You can use the `__file__` attricute for this.